ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -401,5 +539,6 @@ type component_opennebula = {
     'host_ovs'      ? boolean
     'host_hyp'      : string = 'kvm' with match (SELF, '^(kvm|xen)$')
     'tm_system_ds'  ? string with match(SELF, "^(shared|ssh|vmfs)$")
+    'v5_config'     ? boolean
```

Comment:
the `opennebula-server` rpm ships a file called `/var/lib/one/remotes/VERSION`, which has the version in it. can you check with the one developers if this file can be used to determine the version?